### ISSUE TITLE
wandb.watch() produce nothing [CLI] 

### ISSUE THREAD

**Issue by Sun-Rider** [wandb/client#2749]
**Description**

I'm working on encoder-decoder like architecture, with multi-models (i.e., e_1, d_1, e_2, d_2, and e_3, d_3). At the moment, I'm only interested in the gradients flow of the e_3 and d_3, so that I call the wandb.watch() with the following command:
```python
        self.wandb.watch(models=[self.model.module.encoder_3, self.model.module.decoder_3],
                         log='gradients',
                         log_freq=100, log_graph=False)
```
before the training start.

However, I cannot find the "gradients" section in the board. 
![image](https://user-images.githubusercontent.com/33716428/136659466-d057f0c1-ed2b-4c89-b81e-a530e28ed9d2.png)

In addition, wandb version that I used is the latest one at the moment, wandb-0.12.4. 
Is there any thing I did wrong? 
I appreciate all the help !!!

**Environment**
- OS: Ubuntu 18.04
- Environment: Pytorch 1.9, Driver Version: 460.91.03, CUDA Version: 11.2 
- Python Version: 3.7

**Comment by xieyipeng** [comment]
Did you solve this problem？
